app/components/ListDivider/ListDivider.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import { View, StyleSheet } from 'react-native';
+import { Colors } from '../../theme/theme';
+
+export default class ListDivider extends Component {
+  render() {
+    const css = StyleSheet.create({
+      divider: {
```

Comment:
Some css is in a different file and others like this is in the same, is that just based on the size of the css for whether the css is located inside or outside the file?